engineering-feedback/feedback-examples.md

```diff
@@ -0,0 +1,46 @@
+
+## Support Batch Receiving with Sessions in Azure Functions Service Bus Trigger
+
+Customer scenario was to receive a total of 250 messages per second from 50 producers with requirement for ordering & minimum latency, using a Service Bus topic with sessions enabled for ordering. According to [Microsoft documentation](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-performance-improvements#prefetching-and-receivebatch), batch receiving is recommended for better performance but this is not currently supported in Azure Functions Service Bus Trigger. The impact (and work around) was choosing containers over Functions. The Acceptance Criteria is for Azure Functions to support Service Bus sessions with batch and non-batch processing for all Azure Functions GA languages.
+
+> This feedback was [created as a feedback](https://github.com/Azure/azure-functions-servicebus-extension/issues/15) with the Azure Functions product group and also followed up internally until addressed.
+
+## Stream Analytics - No support for zero-downtime scale-down
+
+In order to update the Streaming Unit number in Stream Analytics you need to stop the service and wait for minutes for it to restart. This unacceptable by customers who need near real-time analysis​. In order to have a job re-started, up to 2 minutes are needed and this is not acceptable for a real-time streaming solution.
+It would also be optimal if scale-up and scale-down could be done automatically, by setting threshold values that when reached increase or decrease automatically the amount of RU available. This feedback is for customers' request for zero down-time scale-down capability in stream analytics.
+
+In near-real-time stream processing scenarios, customers need to be able to scale their data processing solution to meet demand as it changes. Some of the time, changes in demand can be predicted, but there will often be situations where data flowing into a stream processing system spikes or dips unexpectedly. Customers building PaaS-based stream processing systems have told us that they expect the services being deployed to support zero-downtime scale changes. Currently, Azure Stream Analytics requires a job to be stopped and restarted to execute a planned scale-down. Job restarts can take up to two minutes to complete, which is not acceptable to our customers in near-real-time scenarios.​
+
+Problem Statement: In order to update the "Streaming Unit" number, partners must stop the service and wait until it restarts. The partner needs to be able to update the number without stopping the service.
+
+Desired Experience: Partners should be able to update the Streaming Unit number without stopping the associated service.
+
+> This feedback was created as a high priority feedback and followed up until addressed in December 2019.
+
+## Python Support for Azure Functions
```

Comment:
Each of these feedback examples follow a different format/layout. Would it be more effective to have a consistent structure? I worry that varying formats may lead to confusion.